File: src/main/java/guru/xgm/language/vjass/lang/VjassCommenter.java
package guru.xgm.language.vjass.lang;

import com.intellij.lang.Commenter;
import org.jetbrains.annotations.Nullable;

public class VjassCommenter implements Commenter {
    @Override
    public @Nullable String getLineCommentPrefix() {
        return "//";
    }

    @Override
    public @Nullable String getBlockCommentPrefix() {
        return "/*";
    }

    @Override
    public @Nullable String getBlockCommentSuffix() {
        return "*/";
    }

    @Override
    public @Nullable String getCommentedBlockCommentPrefix() {
        return "/*";
    }

    @Override
    public @Nullable String getCommentedBlockCommentSuffix() {
        return "*/";
    }
}

File: src/main/kotlin/raft/war/language/vjass/lang/VjassCommenter.kt
package raft.war.language.vjass.lang

import com.intellij.lang.Commenter

class VjassCommenter : Commenter {
    override fun getLineCommentPrefix(): String = "//"
    override fun getBlockCommentPrefix(): String = "/*"
    override fun getBlockCommentSuffix(): String = "*/"
    override fun getCommentedBlockCommentPrefix(): String = "/*"
    override fun getCommentedBlockCommentSuffix(): String = "*/"
}
